automated_testing/test/pageobjects/index-page.ts

```diff
@@ -123,6 +119,22 @@ class IndexPage extends Page {
         return $(selectors.pageSeePastContestsButtonSelector);
     }
 
+    public get secondCompeteCardButtonActiveContests() {
+        return $(selectors.pageAllActiveCardsSelector).$$(selectors.pageCompeteButtonIdSelector)[1];
```

Comment:
Isn't this the same logic as in `competeButtonInActiveContestCard()`?
Maybe just do `competeButtonInActiveContestCard()[1]`